spacy/ml/models/entity_linker.py

```diff
@@ -20,7 +20,7 @@ def build_nel_encoder(
         token_width = tok2vec.maybe_get_dim("nO")
         output_layer = Linear(nO=nO, nI=token_width)
         model = (
-            ((tok2vec >> list2ragged()) & build_span_maker())
+            ((tok2vec >> list2ragged()) & (build_span_maker() >> list2ragged()))
```

Comment:
I think you'd need to increase the architecture version for this kind of change?

Reply:
Ah, that's right, this will break serialization. This is cleaner, but let me see if I can use `list2ragged` internally to avoid a version bump, since this is mainly a bugfix.

Reply:
OK, I have made some simple changes so that the architecture is unchanged and this does not require a version increase.